lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -148,21 +150,25 @@
                         endDate = new Date(end);
 
                     if (pacingType === 'self_paced') {
-                        dateString = 'Self-paced';
-                        if (start && startDate > now) {
-                            dateString += ' - Starts ' + start;
+                        dateString = gettext('(Self-paced) ');
+                        if (start) {
+                            dateString += startDate > now ?
+                                StringUtils.interpolate(gettext('Starts {start}'), {start: start}) :
+                                StringUtils.interpolate(gettext('Started {start}'), {start: start});
                         } else if (end && endDate > now) {
-                            dateString += ' - Ends ' + end;
+                            dateString += StringUtils.interpolate(gettext('Ends {end}'), {end: end});
                         } else if (end && endDate < now) {
-                            dateString += ' - Ended ' + end;
+                            dateString += StringUtils.interpolate(gettext('Ended {end}'), {end: end});
                         }
                     } else {
                         if (start && end) {
                             dateString = start + ' - ' + end;
                         } else if (start) {
-                            dateString = 'Starts ' + start;
+                            dateString += startDate > now ?
+                                StringUtils.interpolate(gettext('Starts {start}'), {start: start}) :
+                                StringUtils.interpolate(gettext('Started {start}'), {start: start});
                         } else if (end) {
-                            dateString = 'Ends ' + end;
+                            dateString += StringUtils.interpolate(gettext('Ends {end}'), {end: end});
```

Comment:
A few notes about the translations:
1.) We should avoid building translated strings programmatically like this, as it can be very difficult (or even impossible) to translated them in fragments. Instead, we should translate the entire string in one call to gettext.
2.) Dates may need to be localized, as they're formatted differently in different languages

Reply:
The specs in the docs say to do it this way - http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#javascript-files